src/modules/showMovies.js

```diff
@@ -0,0 +1,67 @@
+const movies = [
+  {
+    image: '/images/movie1.jpg',
+    ratings: 3,
+    title: 'Hard Target',
+  },
+  {
+    image: '/images/movie1.jpg',
+    ratings: 3,
+    title: 'Hard Target',
+  },
+  {
+    image: '/images/movie1.jpg',
+    ratings: 3,
+    title: 'Hard Target',
+  },
+  {
+    image: '/images/movie1.jpg',
+    ratings: 3,
+    title: 'Hard Target',
+  },
+  {
+    image: '/images/movie1.jpg',
```

Comment:
Great job so far. But according to the requirement please use the data that is coming from the API to dynamically render those `images `and `titles `of the movie